alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/processor/service/HealthProcessorConfigurationImpl.java

```diff
@@ -0,0 +1,115 @@
+package eu.xenit.alfresco.processor.service;
+
+import eu.xenit.alfresco.processor.model.PropertyConverter;
+import lombok.Getter;
+import lombok.Setter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.Properties;
+
+public class HealthProcessorConfigurationImpl implements HealthProcessorConfiguration {
```

Comment:
IMO the overhead of a separate interface is not necessary at this point. 
I also don't like the `...Impl` name of implementations. 

This article seems a good summary of my thoughts (especially paragraph 1.4): 
https://octoperf.com/blog/2016/10/27/impl-classes-are-evil/

Reply:
Or this: https://stackoverflow.com/a/2814831